dracclient/tests/wsman_mocks/physical_disk_view-enum-ok.xml

```diff
@@ -200,7 +200,8 @@
           <n1:UsedSizeInBytes>0</n1:UsedSizeInBytes>
         </n1:DCIM_PhysicalDiskView>
         <n2:DCIM_PCIeSSDView>
-          <n2:BusProtocol>7</n2:BusProtocol>
+    	  <n2:BusProtocol>7</n2:BusProtocol>
+	  <n2:Bus>3E</n2:Bus>
```

Comment:
It looks like the indenting is off on the 2 lines above.  I think they need to be indented 2 more spaces.

Reply:
I opened this file on my dev VM and it shows proper indentation. Also i opened this file on github by doing `view file ` on github and it shows in proper indentation.
will try check by other way too.

Reply:
I just checked, and the issue is that there is a tab in both of these lines.  Can you change it to spaces?

Reply:
Okay thanks for finding out the issue. i will update it in an upstream patch.